util/expiry_os.h

```diff
@@ -40,7 +40,6 @@ class ExpiryModuleOS : public ExpiryModule
     ExpiryModuleOS()
```

Comment:
agreed ... but saving the unit test rework for next branch (which likely starts later today).